lib/scope.js

```diff
@@ -49,6 +50,25 @@ module.exports = {
     return report;
   },  // Ends scope.createFinalReport()
 
+  getSafeScenarioFilename: async function waitForShowIf( scope, { prefix='' }) {
```

Comment:
Does this function really need to wait for show ifs? I think this should say `getSafeScenarioFilename` instead. 

Also, I don't think this function needs to be async either. Nothing in it seems to require waiting for screens/requests.

Reply:
1. `waitForShowIf` isn't even running - it's defining a function name and is a horrible place for that to be. Thank you for spotting it!
2. Many functions in here don't need to be async, but remembering which are and which aren't was causing me a headache, so I decided all `scope` functions would be async. I'm willing to discuss other perspectives, though.